app/src/main/java/swati4star/createpdf/fragment/SplitFilesFragment.java

```diff
@@ -58,6 +60,8 @@
     private BottomSheetUtils mBottomSheetUtils;
     private static final int INTENT_REQUEST_PICKFILE_CODE = 10;
     private BottomSheetBehavior mSheetBehavior;
+    private static final int PERMISSION_REQUEST_WRITE_EXTERNAL_STORAGE_RESULT = 1;
```

Comment:
instead of defining this constant everywhere, let's have one in the PermissionUtils